docs/query.rst

```diff
@@ -286,6 +286,46 @@ Specially, you can filter date part with one of following, note that current onl
     teams = await Team.filter(created_at__month=12)
     teams = await Team.filter(created_at__day=5)
 
+In the PostgreSQL, you can use the ``contains``, ``contained_by`` and ``filter`` options in ``JSONField``:
```

Comment:
MySQL also support that

Reply:
`JSONField` In `mysql` is `text` and this option only available for `postgres`:
```python
class JSONField(Field, dict, list):

    SQL_TYPE = "TEXT"
    indexable = False

    class _db_postgres:
        SQL_TYPE = "JSONB"
    ...
```
So it can not apply on MYSQL.